kubeless.jsonnet

```diff
@@ -0,0 +1,159 @@
+local k = import "ksonnet.beta.1/k.libsonnet";
+local util = import "ksonnet.beta.1/util.libsonnet";
+
+local objectMeta = k.core.v1.objectMeta;
+local deployment = k.apps.v1beta1.deployment;
+local statefulset = k.apps.v1beta1.statefulSet;
+local container = k.core.v1.container;
+local service = k.core.v1.service;
+
+local namespace = "kubeless";
+
+local controllerContainer =
+  container.default("kubeless-controller", "bitnami/kubeless-controller:0.0.13") +
```

Comment:
You're still hardcoding the controller image tag here.  It should somehow be based on `$CONTROLLER_IMAGE`